### PR TITLE
[Easy] make trader diversity spell crosschain - Ready for review 

### PR DESCRIPTION
**Pull request by henrystats**
@gentrexha this is similar to pr: https://github.com/duneanalytics/spellbook/pull/2555

This pr makes the trader diversity spell crosschain

### CODE REVIEW

File: models/labels/trader_dex_diversity/labels_trader_dex_diversity.sql [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+select array(blockchain)                                   as blockchain,
+       address,
+       concat('Number of DEXs traded on: ', dex_diversity) as name,
+       "trader_dex_diversity"                              AS category,
+       "gentrexha"                                         AS contributor,
+       "query"                                             AS source,
+       timestamp('2022-12-15')                             as created_at,
+       now()                                               as updated_at
+from trader_dex_diversity
```
Comment (Hosuke) [inline models/labels/trader_dex_diversity/labels_trader_dex_diversity.sql:33]: Do you want to change `array(blockchain)` to `collect_set(blockchain)`?
https://docs.databricks.com/sql/language-manual/functions/collect_set.html
You can also group by `address` here.
cc. @gentrexha

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @henrystats 

[Sample trader diversity](https://dune.com/queries/1946512?d=1)

File: models/labels/trader_dex_diversity/labels_trader_dex_diversity.sql (hunk 1 above)
Comment (henrystats) [reply models/labels/trader_dex_diversity/labels_trader_dex_diversity.sql:33]: it was originally collect_set(blockchain) but we(with @gentrexha) discussed to make the spells e.g dex diversity grouped by blockchain too so the data is unique for an address that's active across multiple chains. 

So the diversity on ethereum would be different as the one on arbitrum if the trader trades differently.


There have actually been another spell merged (trader age and these ones are similar). cc @soispoke

**Review by jeff-dude — CHANGES_REQUESTED** [review]
please revise after labels overhaul:
https://github.com/duneanalytics/spellbook/pull/2680

more info in the labels directory readme, if necessary

**Comment by henrystats** [comment]
Will update ty ser!

**Pull request closed without merging**